documentation/release_notes.markdown

```diff
@@ -248,6 +320,8 @@ derive data from a PuppetDB export tarball.
 
 * Warn user with unknown config items ([PDB-1067](https://tickets.puppetlabs.com/browse/PDB-1067))
 
+* We now use `honeysql` to generate SQL ([PDB-1228](https://tickets.puppetlabs.com/browse/PDB-1228))
```

Comment:
I think I'll leave this as is, the SQL is still there, just doesn't use somewhat arbitrary spaces that we embed in a string.